nix/sources.json

```diff
@@ -40,5 +40,16 @@
         "type": "tarball",
         "url": "https://github.com/serokell/tezos-packaging/archive/8e7e651e5a02858ffbb9900fb200498777651df6.tar.gz",
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
+    },
+    "morley": {
+        "branch": "master",
+        "description": "Developer tools for the Michelson Language",
+        "homepage": "",
+        "owner": "morley-framework",
+        "repo": "morley",
+        "rev": "f61e31dfd4977350cbc1f3fb23392c8799bc2fe4",
```

Comment:
Erm, it's very sad that now `morley` revision appears in two places. Strictly speaking, it doesn't have to match the one from `stack.yaml`, so it's not that big of a problem, but still sad.
1. Can we somehow extract this information from `stack.yaml` instead?
2. If we can't, I find it better to use `pkgs.haskellPackages.morley`. Right now `morley` on Hackage has an issue that it depends on `tezos-bake-monitor-lib`, but we are very close to eliminating this dependency. Once we finish eliminating it, we will make a new release which will hopefully be usable via `pkgs.haskellPackages.morley`, at least that's how I understand it.

Reply:
> Can we somehow extract this information from stack.yaml instead?

Looks hacky, nix dependencies doesn't have anything to do with stack dependencies

Reply:
https://gitlab.com/morley-framework/morley/-/merge_requests/456

Reply:
Can you try `morley-1.4.0`?